src/sass/index.sass

```diff
@@ -147,6 +152,10 @@
 
     &.sheet
       &.actor
+        top: 0 !important
+        height: 100vh !important
+        max-height: calc(100vh - var(--hotbar-height) - 7px) !important
```

Comment:
Removing 7px makes the sheet not overlap the macro bar slighly which kind of looked odd